src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -150,8 +150,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    help='Applies to all timestamps and durations in a Presentation Time Range, specified as the number of increments in one second.Default is 10000000 - ten million increments in one second, where each increment would be 100 nanoseconds long. For example, if you want to set a startTimestamp at 30 seconds, you would use a value of 300000000 when using the default timescale.')
         c.argument('force_end_timestamp', arg_group='Presentation Time Range', arg_type=get_three_state_flag(),
                    help='Applies to Live Streaming only. Indicates whether the endTimestamp property must be present. If true, endTimestamp must be specified or a bad request code is returned. Allowed values: false, true.')
-        c.argument('bitrate', help='The first quality bitrate.', deprecate_info=c.deprecate(target='bitrate', redirect='firstQuality', hide=True))
-        c.argument('firstQuality', help='The first quality (lowest) bitrate to include in the manifest.')
+        c.argument('bitrate', help='The first quality bitrate.', deprecate_info=c.deprecate(target='bitrate', redirect='first-quality', hide=True))
```

Comment:
Should be: `redirect='--first-quality'` though users would probably get the point.

Reply:
Is it better to also have target="--bitrate" as well? If I set it that way, the message wording changes.
Option '--bitrate' has been deprecated and will be removed in a future release. Use '--first-quality' instead.
Instead of 
Argument 'bitrate' has been deprecated and will be removed in a future release. Use '--first-quality' instead.

Reply:
Yes, good catch!
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments

Target and redirect should both be in options format.

Reply:
Thanks.